src/lib/item-list/list.component.ts

```diff
@@ -54,6 +54,12 @@ export class PlexListComponent implements AfterViewInit {
 
     @Input() size: PlexSize = 'md';
 
+    @Input() set offset(value: number) {
+        if (typeof value === 'number') {
+            this.height = `calc(100% - ${value}px)`;
```

Comment:
Me parece mejor usar `vh` en lugar de `%`ya que toma todo el alto visible. En cambio el porcentaje toma todo el alto, visible o no y suele calcular mal en algunos casos.
```suggestion
            this.height = `calc(100vh - ${value}px)`;
```